pkg/portfwd/client.go

```diff
@@ -24,26 +25,10 @@ func HandleTCPConnection(ctx context.Context, client *guestagentclient.GuestAgen
 		return
 	}
 
-	g, _ := errgroup.WithContext(ctx)
-
 	rw := &GrpcClientRW{stream: stream, id: id, addr: guestAddr}
-	g.Go(func() error {
-		_, err := io.Copy(rw, conn)
-		if errors.Is(err, io.EOF) {
-			return nil
-		}
```

Comment:
For this code, I would start with removing the handling of io.EOF if we think that it is not possible.

Reply:
Eh. What's the benefit of doing that in a separate change? Given that it takes multiple days to get a review I don't see the upside. It is most definitely not possible, it's right there in the docs.

Reply:
If you post a trivial change like this it will be merged pretty quickly. People have limited time to review, and it is much easier to review tiny changes. Changes like this one, changing multiple components are hard to review.